packages/cosmos_ui_components/lib/components/cosmos_elevated_button.dart

```diff
@@ -30,12 +32,16 @@ class CosmosElevatedButton extends StatelessWidget {
       child: Row(
         mainAxisSize: MainAxisSize.min,
         children: [
+          if (prefixIcon != null) ...[
+            prefixIcon!,
+            SizedBox(width: CosmosTheme.of(context).spacingM),
+          ],
           Text(
             text,
             style: CosmosTextTheme.elevatedButton,
           ),
           if (suffixIcon != null) ...[
-            SizedBox(width: CosmosTheme.of(context).spacingS),
+            SizedBox(width: CosmosTheme.of(context).spacingM),
```

Comment:
this changes current design for other places, if you need other spacing in a particular case then lets parametrize it with the default value of `spacingS`

Reply:
Updated